PanModal/Controller/PanModalPresentationController.swift

```diff
@@ -759,7 +764,8 @@ private extension PanModalPresentationController {
      Halts the scroll of a given scroll view & anchors it at the `scrollViewYOffset`
      */
     func haltScrolling(_ scrollView: UIScrollView) {
-        scrollView.setContentOffset(CGPoint(x: 0, y: scrollViewYOffset), animated: false)
+        guard presentable?.shouldHaltScroll != false else { return }
```

Comment:
again, no need to check if something is not false. 

My suggestion: 
`if presentable?.shouldHaltScroll { return }`
which reads a bit easier 😉